client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -26,6 +26,22 @@ import PluginSettingsLink from 'my-sites/plugins/plugin-settings-link';
 import PluginInformation from 'my-sites/plugins/plugin-information';
 import { userCan } from 'lib/site/utils';
 
+export function getAvailableNewVersions( props ) {
```

Comment:
I don't quite understand what this function does. My guess is that it returns the latest new version number of a plugin. 

I think it would really help if we would be more granular what we pass to it. For example the just the list sites and not the whole props of an object.